kore/src/Kore/Strategies/Verification.hs

```diff
@@ -95,6 +103,22 @@ didn't manage to verify a claim within the its maximum number of steps.
 If the verification succeeds, it returns ().
 -}
 
+data StuckVerification patt claim
+    = StuckVerification
+        { stuckPattern :: !patt
```

Comment:
Is it possible to actually return the current proof state here, or only the pattern we got stuck on?